### PR TITLE
docs - update signed embedding

### PR DESCRIPTION
**Pull request by nllho**
Some clarifications based on recent customer questions.

### CODE REVIEW

File: docs/embedding/signed-embedding-parameters.md [hunk 1]
```diff
@@ -54,22 +55,23 @@ If the original dashboard's filter widget accepts multiple values (i.e., it's a
...
+
+You can use locked parameters to display filtered data based on attributes captured by your web server (such as a user's login). For more examples, see the [reference apps repo](https://github.com/metabase/embedding-reference-apps).
 
 1. Go to your dashboard or SQL question. Make sure you've set up a [dashboard filter](../dashboards/filters.md) or [SQL variable](../questions/native-editor/sql-parameters.md).
 2. Click on the **sharing icon** > **Embed this item in an application**.
 3. Under **Parameters**, you'll find the names of your dashboard filters or SQL variables.
 4. Select **Locked** for each parameter that should pre-filter your data.
 5. Add values for the filter under **Preview locked parameters**.
 6. Click **Publish** to save your changes.
+7. Add or update the code on your server so that it matches the code generated by Metabase (Code > top block).
```
Comment (jeff-bruemmer) [inline docs/embedding/signed-embedding-parameters.md:70]:
```suggestion
7. Add or update the code on your server to match the code generated by Metabase (Code > top block).
```

Can you explain what you mean by "Code > top block"?

File: docs/embedding/signed-embedding-parameters.md [hunk 2]
```diff
@@ -54,22 +55,23 @@ If the original dashboard's filter widget accepts multiple values (i.e., it's a
...
-
-Note that you can only add filter values that match the filter type on the _original_ dashboard or SQL question. For example, if you have a text box filter on your original dashboard or SQL question, you'll only be able to add a single filter value to your locked parameter. If you want to provide multiple filter values to the parameter, you'll need to change the original filter to a [dropdown filter](../dashboards/filters.md#choosing-between-a-dropdown-or-autocomplete-for-your-filter) first.
+Note that you can only add filter values that match the filter type on the original dashboard or SQL question. For example, let's say you have a text box filter on your original dashboard called "Test" with the values "A", "B", and "Control". With this setup, you'll be able to choose _one_ of "A", "B", or "Control" for your locked parameter. If you want to provide multiple filter values to the parameter, like "A" _and_ "B", you'll need to change the original filter to a [dropdown filter](../dashboards/filters.md#choosing-between-a-dropdown-or-autocomplete-for-your-filter) first.
```
Comment (jeff-bruemmer) [inline docs/embedding/signed-embedding-parameters.md:74]: Good note here. Maybe a heading to aid skimming and to highlight the main point? Something like: 

"To filter multiple values, change the filter type to a dropdown."

**Review by jeff-bruemmer — APPROVED** [review]

**Comment by jeff-bruemmer** [comment]
Thanks, @nllho!

File: docs/embedding/signed-embedding-parameters.md (hunk 1 above)
Comment (nllho) [reply docs/embedding/signed-embedding-parameters.md:70]: Yeah, it looks like this:

publish-dashboard

**Pull request merged**